src/file.cpp

```diff
@@ -412,38 +412,43 @@ static xmlNodePtr find_section_start_node(const std::string& file)
     return section;
 }
 
-static wf::config::config_manager_t load_xml_files(const std::string& xmldir)
+static wf::config::config_manager_t load_xml_files(const std::vector<std::string>& xmldirs)
 {
-    auto xmld = opendir(xmldir.c_str());
-    if (NULL == xmld)
-    {
-        LOGE("Failed to open XML directory ", xmldir);
-        return {};
-    }
-
-    LOGI("Reading XML configuration options from directory ", xmldir);
     wf::config::config_manager_t manager;
-    struct dirent *entry;
-    while ((entry = readdir(xmld)) != NULL)
+
+    for (auto& xmldir : xmldirs)
     {
-        if (entry->d_type != DT_LNK && entry->d_type != DT_REG)
-            continue;
+        auto xmld = opendir(xmldir.c_str());
+        if (NULL == xmld)
+        {
+            LOGE("Failed to open XML directory ", xmldir);
+            return {};
```

Comment:
I suppose in case we have multiple XML directories we wouldn't want to fail if one of them is not readable, but just print the error and continue on to the next one?